lib/improver/spotdata/extract_data.py

```diff
@@ -207,28 +227,41 @@ def make_cube(self, cube, data, sites):
             at the spotdata sites.
 
         """
-        time_coord = cube.coord('time')
-        time_coord.convert_units('hours since 1970-01-01 00:00:00')
+        # Ensure leading dimension axis is time.
```

Comment:
I've changed this. I only need to ensure that time is a dimension coordinate, it needn't be leading.